HintList.py

```diff
@@ -1961,6 +1975,11 @@ def rainbow_bridge_hint_kind(world: World) -> str:
         'location_fallback': '\x05\x42\x06\x3dForest Stage\x04\x01\x05\x40\x06\x14We are waiting to see your\x01\x06\x32beautiful face!\x01\x06\x28Win fabulous prizes!',
         'text_style': 0x13,
     },
+    ('ocarina_of_time', 'song_of_time'): {
+        'id': 0x707A,
+        'location_text': 'It is also written that reuniting \x05\x41The Spiritual Stones\x05\x40 leads to \x05\x42{item_1}\x05\x40 and \x05\x42{item_2}\x05\x40',
```

Comment:
The existing altar text is written as if you're reading it directly from the altar, so I think “it is also written” should be removed.

```suggestion
        'location_text': 'Reuniting the \x05\x41Spiritual Stones\x05\x40 leads to \x05\x42{item_1}\x05\x40 and \x05\x42{item_2}\x05\x40.',
```